edx_sysadmin/views.py

```diff
@@ -74,10 +66,10 @@ def post(self, request):
             try:
                 course = get_course_by_id(course_key)
                 course_found = True
-            except Exception as err:  # pylint: disable=broad-except, translation-of-non-string
+            except Exception as err:  # pylint: disable=broad-except
                 self.msg += _(  # pylint: disable=translation-of-non-string
                     HTML(
-                        u'<div class="error">Error - cannot get course with ID {0}<br/><pre>{1}</pre></div>'
+                        '<div class="error">Error - cannot get course with ID {0}<br/><pre>{1}</pre></div>'
```

Comment:
mark strings for translation which are being used in HTML like this
```
${Text(_("{div_start}Error - cannot get course with ID {course_key}{br}{pre_start}{err_msg}{pre_end}{div_end}.")).format(
    div_start=HTML('<div class="error">'),
    div_end=HTML('</div>'),
    br=HTML('<br />'),
    course_key=course_key
    .............
    .............
)}
```
see the i18n guide for details
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files

Reply:
updated.